src/EnergyPlus/ElectricBaseboardRadiator.cc

```diff
@@ -121,13 +121,12 @@ namespace ElectricBaseboardRadiator {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int BaseboardNum; // Index of unit in baseboard array
         auto &ElecBaseboard = state.dataElectBaseboardRad->ElecBaseboard;
-        auto &GetInputFlag = state.dataElectBaseboardRad->GetInputFlag;
```

Comment:
auto const &GetInputFlag?

Reply:
GetInputFlag is modified in this function so it can't be const. Also should not use auto for scalars, so it shouldn't be auto. There is no need for a shortcut for this bool.

Reply:
Yes, all shortcuts that fit this category are simply being removed.

Reply:
oh, right right